docs/source/usage.rst

```diff
@@ -113,9 +113,12 @@ Peeling arguments
                             each peeling cycle.
       -est_rec_prob         Flag to re-estimate the recombination rates after
                             each peeling cycle.
-      -est_alt_allele_prob  Flag to re-estimate the alternative allele probabilities for each metafounder after
-                            each peeling cycle.
-      -no_phase_founder    A flag phase a heterozygous allele in one of the
+      -est_alt_allele_prob  Flag to estimate the alternative allele frequencies using
+                            all observed genotypes prior to peeling.
+      -update_alt_allele_prob
```

Comment:
@RosCraddock we will probably experiment a bit with this. I like and dislike the estimate and update options we have here. It’s a good approach I think nevertheless. The dislike part is that it’s another option, but it’s ok - it gives us flexibility.

At the moment the “estimation” uses the “Newton” path in the code on observed genotypes anywhere in the pedigree, but only with one metafounder. The “update” uses “EM” path in the code on estimated genotype probabilities in founders, for any number of metafounders.

Maybe we find a way to expand the “Newton” path, so indeed best to keep the options separate.

But, to make it lighter on users should we activate estimation when we set the updating? Hmm, I guess not, because current default is to use 0.5, which would not be activated at all. Ok, I just had to write this up to clear everything in my head;) All good, what we have is very good set of options.